spec/facility_spec.rb

```diff
@@ -23,4 +24,62 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe 'add services to facility' do
+    before(:each) do
+      @facility_1 = Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
+      @facility_2 = Facility.new({name: 'DMV Northeast Branch', address: '4685 Peoria Street Suite 101 Denver CO 80239', phone: '(720) 865-4600'})
+      @cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+      @bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+      @camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice} )
+      end
+
+    it 'can add services to a facility' do
+      expect(@facility_1.add_service('Vehicle Registration')).to eq(["Vehicle Registration"])
+    end
+
+    it 'can read registration date' do
+      expect(@cruz.registration_date).to eq(nil)
+    end
```

Comment:
I don't think this needs to be its own test, though we might want to include the line as part of the `#register_vehicle` test later on